CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contribution Guidelines
+
+**Note:** If these contribution guidelines are not followed your issue or PR might be closed, so
+please read these instructions carefully.
+
+_See also: [Flutter's code of conduct](https://flutter.dev/design-principles/#code-of-conduct)_
+
+## About
+
+Melos is a tool that optimizes the workflow around managing multi-package repositories with git and Pub.
+
+## Contribution types
+
+### Bug Report
+
+- If you find a bug, please first report it using [Github issues](https://github.com/invertase/melos/issues/new?assignees=&labels=bug%2Ctriage&template=bug_report.yml&title=fix%3A++).
+  - First check if there is not already an issue for it; duplicated issues will be closed.
+
+### Bug Fix
+
+- If you'd like to submit a fix for a bug, please read the [How To](#how-to-contribute) for how to send a Pull Request.
+- Indicate on the open issue that you are working on fixing the bug and the issue will be assigned to you.
+- Write `Fix #xxxx` in your PR text, where xxxx is the issue number (if there is one).
+- Include a test that isolates the bug and verifies that it was fixed.
+
+### New Features
+
+- If you'd like to add a feature to the library that doesn't already exist, feel free to describe the feature in a new [GitHub issue](https://github.com/invertase/melos/issues/new?assignees=&labels=bug%2Ctriage&template=bug_report.yml&title=fix%3A++).
+- If you'd like to implement the new feature, please wait for feedback from the project maintainers before spending too much time writing the code. In some cases, enhancements may not align well with the project future development direction.
+- If applicable, implement the code for the new feature and please read the [How To](#how-to-contribute).
+
+### Documentation & Miscellaneous
+
+- If you have suggestions for improvements to the documentation or examples (or something else), we would love to hear about it.
+- As always first file a [Github issue].
```

Comment:
```suggestion
- As always first file a [GitHub issue](https://github.com/invertase/melos/issues/new).
```

@blaugold There is no issue template for this kind of issue, right?

Reply:
That's correct. If you want to add one... :)